### ISSUE TITLE
Will there be a new release soon?

### ISSUE THREAD

**Issue by palladia** [microsoft/ApplicationInsights-Go#53]
Azure pipeline component detections is raising a major security issue due to ApplicationInsights-Go including satori/uuid.go.  I noticed that it's been fixed recently, but will there be a new release soon to address this issue?  Version 0.4.4?

**Comment by jjjordanmsft** [comment]
Yes, I am looking into whether I can get another long-wanted change in as well.

For the record, I've been aware of the `satori/uuid.go` bug for [a while](https://github.com/microsoft/ApplicationInsights-Go/issues/20#issuecomment-387253730) [now](https://github.com/microsoft/ApplicationInsights-Go/commit/907a0d61fb8cf6c46b94590bd84f23455e4fe0d3#diff-114112a65d6d0888757db19ba5dbb5e28c68fb42ed00ab8e689696b4a8dbd99dR50) and the underlying issue that's flagging this doesn't affect 0.4.3.  This won't satisfy an automated tool, but I feel is worth noting.

**Comment by cjheppell** [comment]
Is there an update on this?

Whilst I appreciate that you've identified this library isn't vulnerable to the identified issue, we use a vulnerability scanning tool as part of our CI/CD process and as such, this library is getting identified as vulnerable. This is causing some pain in our release process, as we've having to manually resolve this vulnerability each deployment.

A new release that contains the updated library would resolve this problem entirely.

**Comment by jjjordanmsft** [comment]
Understood, I'll roll one by EOW. Thanks for your patience.

**Comment by jjjordanmsft** [comment]
Done. Let me know if this mollifies your CI/CD pipelines.
